docs-gen/content/docs/reference/configuration.md

```diff
@@ -86,9 +87,9 @@ The configured `serviceAccount` needs to have [workload identity](https://cloud.
   - `storage.objects.create` on the configured `pipelineBucket`
   - `storage.objects.get` on the configured `pipelineBucket`
   - `storage.objects.delete` on the configured `pipelineBucket`
-  - `cloudscheduler.jobs.create`
-  - `cloudscheduler.jobs.update`
-  - `cloudscheduler.jobs.delete`
+  - deprecated - `cloudscheduler.jobs.create`
+  - deprecated - `cloudscheduler.jobs.update`
+  - deprecated - `cloudscheduler.jobs.delete`
```

Comment:
don't we also need permission to interact with the VAI scheduler API?

Reply:
Correct, not sure why I thought the role had been listed. Will add

Reply:
Done